user_docs/en/changes.t2t

```diff
@@ -63,6 +63,7 @@ What's New in NVDA
 Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html#API] for information on NVDA's API deprecation and removal process.
 
 - Instantiating ``winVersion.WinVersion`` objects with unknown Windows versions above 10.0.22000 such as 10.0.25398 returns "Windows 11 unknown" instead of "Windows 10 unknown" for release name. (#15992, @josephsl)
+- Make the Appveyor compilation process easier for private NVDA builds, by using variables in appveyor.yml to disable or modify portions of the build scripts which are otherwise difficult for those outside NV Access to run in their default form. (#16216, @XLTechie)

```

Comment:
General rephrasing as forks are often public, not private. "Personal" might be better here, but there are also organisational forks like NVDA JP.

```suggestion
- Make the AppVeyor build process easier for NVDA forks, by adding configurable variables in appveyor.yml to disable or modify NV Access specific portions of the build scripts. (#16216, @XLTechie)
```